applications/tari_validator_node/src/bootstrap.rs

```diff
@@ -356,13 +359,16 @@ where
         // Create the second layer tari resource
         let mut metadata = Metadata::new();
         // TODO: decide on symbol for L2 tari
-        metadata.insert(TOKEN_SYMBOL, "tXTR2".to_string());
+        // metadata.insert(TOKEN_SYMBOL, "tXTR2".to_string());
 
         tx.insert_substates(SubstateShardData::new(
             shard_id,
             address,
             0,
-            Substate::new(0, Resource::new(ResourceType::Confidential, metadata)),
+            Substate::new(
+                0,
+                Resource::new(ResourceType::Confidential, "tXTR2".to_string(), metadata),
```

Comment:
What does `tXTR2` stand for ?

Reply:
It's the testnet layer 2 tari. No official name yet. 

ObjectClaim was an idea I had in an earlier design. It was to prove ownership on a per-shard level. It turns out it's easier to use the whole transaction's ownership, so it's better to just use that